Widescreen patch
Forest, intro chase cutscene
Tree
New: Do not hide tree
--- original
+++ modified
@@ -45,7 +45,6 @@
 rel :JMP_8c_0
 JMP_8c_0:
 call 111, 0x1
# This tree normally starts hidden
-call 106, 0x1
 call 101, stor[0], 0xfffffe8e, 0x23, 0x214
 call 103, stor[0], 0x0, 0xc8, 0x0
 call 97, 0x0
@@ -70,6 +69,7 @@
 call 691, stor[0], 0x13
 gosub inl[:LABEL_0]
 call 99, 0x1
+rewind
 mov 0x0, stor[8]
 LABEL_4:
 call 691, stor[0], stor[8]
